src/report/generate_https_scan_report.py

```diff
@@ -0,0 +1,926 @@
+#!/usr/bin/env python3
+
+"""Create Cyber Hygiene HTTPS Report PDF.
+
+Usage:
+  generate_https_scan_report [options] "AGENCY"
+  generate_https_scan_report (-h | --help)
+  generate_https_scan_report --version
+
+Options:
+  -d --debug                     Keep intermediate files for debugging.
+  -h --help                      Show this screen.
+  --version                      Show version.
+"""
+# Standard Python Libraries
+import codecs
+import csv
+from datetime import datetime
+import json
+import os
+import shutil
+
+# The subprocess module is frowned upon by Bandit, but we need it
+# here.
+import subprocess  # nosec B404
+import sys
+import tempfile
+
+# Third-Party Libraries
+import chevron
+from docopt import docopt
+
+# intra-project modules
+import graphs
```

Comment:
I prefer absolute imports for things like this.

```suggestion
from . import graphs
```

Reply:
This change is rejected at runtime due to this code not being in an actual package.  Can I revert the change, or would prefer that I try to turn the `report` package into a proper Python package.

Reply:
In the interest of getting this out the door, I'd vote to revert and create an issue to make `report` a proper package at a later date.

Reply:
@mcdonnnj - Please see issue #50 and let me know if this is sufficient for your re-approval.  See also commit fa8f6e3.